owncloudData/src/main/java/com/owncloud/android/data/files/datasources/LocalFileDataSource.kt

```diff
@@ -46,6 +45,7 @@ interface LocalFileDataSource {
     fun getFilesAvailableOfflineFromEveryAccount(): List<OCFile>
     fun getFileWithSyncInfoByIdAsFlow(id: Long): Flow<OCFileWithSyncInfo?>
     fun moveFile(sourceFile: OCFile, targetFolder: OCFile, finalRemotePath: String, finalStoragePath: String)
+    fun copyFile(sourceFile: OCFile, targetFolder: OCFile, finalRemotePath: String, remoteId: String, replace: Boolean?)
```

Comment:
Will the order of the functions in the classes be alphabetical or by case?

Reply:
Not alphabetical but semantical, in this case we have first all the `get...` and then other operations with files, so the copy method didn't fit in the first place